web/src/installers/versions.js

```diff
@@ -0,0 +1,210 @@
+
+// first version of each is "latest"
+module.exports.InstallerVersions = {
+  kubernetes: [
+    "1.19.7",
+    "1.19.3",
+    "1.19.2",
+    "1.18.10",
+    "1.18.9",
+    "1.18.4",
+    "1.17.13",
+    "1.17.7",
+    "1.17.3",
+    "1.16.4",
+    "1.15.3",
+    "1.15.2",
+    "1.15.1",
+    "1.15.0",
+    "1.20.4", "1.20.2", "1.20.1", "1.20.0", // cron-kubernetes-update
```

Comment:
the autoupdate scripts will all need to be updated to point at this filepath and not the previous one

Reply:
updated